site/docs/components/navigation-item/accessibility.mdx

```diff
@@ -10,56 +10,54 @@ data:
 
 ### Keyboard interactions
 
-#### Horizontal Navigation
+#### Horizontal navigation
 
 <KeyboardControls>
   <KeyboardControl keyOrCombos="Tab">
   
-  - Moves focus into the first Navigation Item in the group. Focus is set to the entire component.
+  - Tab moves focus into the first navigation item in the group. Sets focus to the entire component.
   
-  - When the Navigation Item has children, tab again, to move focus on to the arrow icon.
+  - When the navigation item has children, second key press moves focus on to the arrow icon.
```

Comment:
This is technically incorrect, although it was incorrect previously. The arrow icon never receives focus